tests/io/v3/vm/agent_exclusions/test_agent_exclusions.py

```diff
@@ -0,0 +1,364 @@
+TIMEZONE_URL: str = 'https://cloud.tenable.com/api/v3/scans/timezones'
+
+
+@responses.activate
+def test_create(api):
+    '''
+    Test case for agent_exclusion create method
+    '''
+    name: str = 'Example Weekly Exclusion'
+    description: str = 'This is a weekly exclusion example'
+    enable: bool = True
+    frequency: str = 'WEEKLY'
+    interval: int = 1
+    weekdays: list = ['MO', 'WE', 'FR']
+    start_time: str = datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S')
+    end_time: str = (
+            datetime.utcnow() + timedelta(hours=1)
+    ).strftime('%Y-%m-%d %H:%M:%S')
+    timezone: str = 'Etc/UTC'
+
+    # Let's create test response for create agent exlusion endpoint
+    test_response = {
+        'description': description,
+        'name': name,
+        'id': '534d4121-b55f-465c-bf79-0ff34d0b9a88',
+        'creation_date': '2020-04-21T10:51:47+00:00',
+        'last_modification_date': '2020-04-21T10:51:47+00:00',
+        'core_updates_blocked': True,
+        'schedule': {
+            'starttime': '2020-04-21T20:30:00+07:00',
+            'endtime': '2020-04-21T21:00:00+07:00',
+            'enabled': True,
+            'rrules': {
+                'freq': 'ONETIME',
+                'interval': '1'
+            },
+            'timezone': timezone
+        }
+    }
+
+    # Let's create sample payload for create agent exclusion endpoint
+    payload = {
+        'name': name,
+        'description': description,
+        'schedule': {
+            'enabled': enable,
+            'starttime': start_time,
+            'endtime': end_time,
+            'timezone': timezone,
+            'rrules': {
+                'freq': frequency,
+                'interval': interval,
+                'byweekday': ','.join(weekdays)
+            }
+        }
+    }
+
+    # Let's register the response for timezone API
+    responses.add(
+        responses.GET,
+        TIMEZONE_URL,
+        json={
+            'timezones': [
+                {
+                    'name': 'Africa/Addis_Ababa',
+                    'value': 'Africa/Addis_Ababa'
+                },
+                {
+                    'name': 'Etc/UTC',
+                    'value': 'Etc/UTC'
+                }
+            ]
+        }
+    )
+
+    # Let's mock the response for create agent exclusion API endpoint
+    responses.add(
+        responses.POST,
+        re.compile(f'{BASE_URL}'),
+        match=[responses.matchers.json_params_matcher(payload)],
+        json=test_response
+    )
+
+    res = api.v3.vm.agent_exclusions.create(
+        frequency=frequency,
+        name=name,
+        start_time=start_time,
+        enabled=enable,
+        end_time=end_time,
+        description=description,
+        interval=interval,
+        weekdays=weekdays
+    )
+
+    assert isinstance(res, dict)
+    assert res['name'] == name
+
+
+@responses.activate
+def test_delete(api):
+    '''
+    Test case for agent_exclusion delete method
```

Comment:
Correct text `agent_exclusion` to `agent_exclusions ` need to update for many places

Reply:
As discussed on last code, no need to change